recipe/meta.yaml

```diff
@@ -79,10 +77,9 @@ test:
     - pip
     - wheel
     - tbb >=2021.6,<2022
-    - llvm-openmp              # [osx]
```

Comment:
Curiosity question: What changed allowing this to be removed? When I read setup.py there isn't much recent change around the OpenMP requirements for darwin.

Reply:
It's redundant as llvm-openmp is already a run dependency of numba (through the implicit strong run export triggered by its inclusion in the build deps).